### DISCUSSION TITLE
Project Problem 3

### DISCUSSION THREAD

**Discussion by courtcarroll** [cooperrc/computational-mechanics#121] [Q&A]
Can anyone help me with this code for problem 3 
<img width="1440" alt="Screen Shot 2022-10-19 at 4 50 18 PM" src="https://user-images.githubusercontent.com/112587455/196801961-4341319f-8e90-4b93-b0c3-1dc7c25fb326.png">
<img width="890" alt="Screen Shot 2022-10-19 at 4 50 22 PM" src="https://user-images.githubusercontent.com/112587455/196801973-f9ac9b01-2482-4042-afd6-a72d50a174ee.png">

**Comment by cooperrc** [comment]
there are 2 things that could be going wrong:
1. the value of `dx` is too high, so try using `mod_secant(f_dm, 0.001, 0.05)`
2. _more likely_: the function `f_dm` might have a typo/mistake that leads to mass<=0 and you're getting an error when acceleration gets calculated as `c*v**2/0`
